products/postgresql.md

```diff
@@ -25,113 +25,119 @@ identifiers:
 -   purl: pkg:docker/library/postgres
 
 releases:
+-   releaseCycle: "16"
+    releaseDate: 2023-09-14
+    eol: false
```

Comment:
```suggestion
    eol: 2028-11-09
```

Final release expected on November 9, 2028 according to https://www.postgresql.org/support/versioning/.